src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -660,11 +660,15 @@ private void removeTerritoriesWhereTransportsAreExposed() {
             ProBattleUtils.calculateBattleResults(player, t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data),
                 patd.getBombardTerritoryMap().keySet(), true);
         int production = 0;
+        int isEnemyCapital = 0;
         final TerritoryAttachment ta = TerritoryAttachment.get(t);
         if (ta != null) {
           production = ta.getProduction();
+          if (ta.isCapital()) {
+            isEnemyCapital = 1;
```

Comment:
Would it not be simpler to just set this to 3, rather than using the 'magic number' scaling factor later? If the scaling factor were labelled as such, I would have less objection.

More importantly though, how do you know that this puts enough weight on launching amphib attacks on capitols? It does seem like we are missing the value of the IPC you would capture, and some more value for the strategic benefit of denying production for a turn as well.

Last, I think there should be yet even more weight added if the capitol is not recapturable. Given where things are, adding that consideration in the future I think is reasonable, but in the meantime I would tend to think some additional factors have not been added in to increase the weight even more.